install_config/router/f5_router.adoc

```diff
@@ -23,9 +23,8 @@ endif::[]
 
 [WARNING]
 ====
-The F5 router plug-in will be deprecated in {product-title} version 3.11. It
-will be replaced by the F5 BIG-IP Controller. For more information, see link:http://clouddocs.f5.com/containers/v2/openshift/[F5 Container Connector - OpenShift]. For information about migrating to the BIG-IP Controller, see
-link:http://clouddocs.f5.com/containers/v2/openshift/replace-f5-router.html[Replace the F5 Router in OpenShift].
+The F5 router plug-in will be deprecated in {product-title} version 3.11. The functionality of the F5 router plug-in has been replaced in the F5 BIG-IP® Controller for OpenShift, and it is now recommended for greenfield deployments. For more information, see link:http://clouddocs.f5.com/containers/v2/openshift/[F5 BIG-IP Controller for OpenShift]. For information about migrating existing deployments from the F5 router plug-in to the BIG-IP Controller for OpenShift, see
```

Comment:
s/has been replaced/is
s/OpenShift, and it is now recommended for greenfield deployments/OpenShift. Use that plug-in for new clusters. (We don't use "recommend" or jargon like "greenfield" when it's possible to state it another way.)
s/F5 BIG-IP Controller for OpenShift/F5 Container Connector - OpenShift (We need to use the actual link title.)
s/For information about migrating existing deployments from the F5 router plug-in to the BIG-IP Controller for OpenShift/To migrate an existing cluster from the F5 router plug-in to the BIG-IP Controller for OpenShift
s/Replace the F5 Router with the F5 BIG-IP Controller for OpenShift in OpenShift/Replace the F5 Router in OpenShift

Reply:
@kalexand-rh On this one s/F5 BIG-IP Controller for OpenShift/F5 Container Connector - OpenShift (We need to use the actual link title.) We had initially called this the container connector, but have had a branding change to call it the "F5 BIG-IP Controller for OpenShift" not all docs are updated yet on our side but I am going through similar update PRs for docs on our side as well. I'll go with whatever you decide here and make another PR to this repo when the link title changes on our side if you wish.

All other suggestions I'm fine with.

Reply:
@pdp2shirts, I'm ok with changing to what you expect the title to be to avoid double work.